python/ray/dashboard/metrics_exporter/actions.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import typing
+
+from ray.dashboard.metrics_exporter.schema import ActionType, KillAction
+
+logger = logging.getLogger(__name__)
+
+
+class ActionHandler:
+    def __init__(self, dashboard_controller):
+        self.dashboard_controller = dashboard_controller
+
+    def handle_kill_action(self, action: dict):
+        kill_action = KillAction.parse_obj(action)
+        self.dashboard_controller.kill_actor(
+            kill_action.actor_id, kill_action.ip_address, kill_action.port)
+
+    def handle_actions(self, actions: typing.List[dict]):
+        for action in actions:
+            action_type = action.get("type", None)
+
+            if action_type == ActionType.KILL_ACTOR:
+                self.handle_kill_action(action)
+            else:
+                continue
```

Comment:
this should raise NotImplemented

Reply:
One problem of raising an error is that actions that are behind the NotImplementedError won't be executed. What about just adding error logs?

Reply:
hmmm maybe? our server log is very very cluttered and i don't think we can start with it anytime soon